src/Batch/Batch/Models/PSPrivateLinkServiceConnectionState.cs

```diff
@@ -58,7 +58,7 @@ internal static PSPrivateLinkServiceConnectionState CreateFromPrivateLinkService
             return new PSPrivateLinkServiceConnectionState(
                 privateLinkServiceConnectionState.Status,
                 privateLinkServiceConnectionState.Description,
-                privateLinkServiceConnectionState.ActionRequired);
```

Comment:
Breaking changes from `ActionRequired` to `ActionsRequired`

Reply:
Good catch. Fixed. Thank you.